src/main/java/com/soartech/soarls/tcl/TclAstNode.java

```diff
@@ -165,6 +165,30 @@ public void printTree(PrintStream stream, char input[], int depth) {
     }
   }
 
+  /** @return a list containing the node being called, and all of its children recursively * */
+  public List<TclAstNode> flatten() {

```

Comment:
This function never gets used, and it potentially allocates a lot of memory, so I think it's best to get rid of it.